### DISCUSSION TITLE
Random items adding to Highlighted Items, can't delete things from highlighted items etc

### DISCUSSION THREAD

**Discussion by Ashturbate** [runelite/runelite#17903] [Q&A] [answered]
I'm almost certain this might be a bad interaction with another plugin perhaps but random items are getting added to my highlighted items list and whenever I delete items in the list it doesn't save nor can I clear it.

![image](https://github.com/runelite/runelite/assets/37551486/b6a682fa-4b83-4d29-9235-5d8325c3fc37)

Are there any logs I can extract to try and figure this out?

Needless to say getting coins, addy arrows, planks and other random bs highlighted is annoying.

Appreciate this is probably a problem of my own making but any help would be apprecaited.

Worthing noting when I press alt + - (to remove the item) both the + and - get highlighted.

Super strange.

Thank you in advance!

**Comment by geheur** [comment, marked as answer]
it's probably that barbarian assault plugin.

**Comment by Ashturbate** [comment]
Which one sorry, this highlighted one?

![image](https://github.com/runelite/runelite/assets/37551486/7a3ac5cb-75a4-4e7d-8b07-b970a504cd3f)

Ignoring my awful spelling of Barbarian :P

**Reply by geheur** [reply]
yes

**Reply by Ashturbate** [reply]
Okay, thank you - it was installed (have since uninstalled) so will see how I go.

Thank you if this does solve it I've been googling all sorts trying to figure it out!

Deeply sorry if I've missed documentation saying they are incompatible!

**Reply by geheur** [reply]
the plugin is just broken somehow, I think.

**Reply by Ashturbate** [reply]
That's seemingly done the trick. You're legend thank you!
